src/integrationtest/resources/com/epam/idea/rest/controller/tag/tag-entries.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<dataset>
```

Comment:
Add `<user>` entry. U referencing the user with id = 1 in the next line 
```xml
<idea id="1" ... user_id="1"/>
```
but there's no such user in the database and u will get get an error in standard SQL.
There's no error because of the following line in the `test-schema.sql` file:
```sql 
 SET REFERENTIAL_INTEGRITY FALSE
```
We will probably remove this line later.